pkg/registry/registry.go

```diff
@@ -145,3 +146,8 @@ func (or OktetoRegistry) HasGlobalPushAccess() (bool, error) {
 	image := or.imageCtrl.ExpandOktetoGlobalRegistry(globalTestImage)
 	return or.client.HasPushAccess(image)
 }
+
+// GetImageCtrl returns the image controller
+func (or OktetoRegistry) GetImageCtrl() cache.ImageCtrlInterface {
```

Comment:
I find odd to return an interface from another package. It should return the ImageCtrl from this package and the clients implement their own interface

Reply:
we resolve this together and merged the other PR into this branch